### PR TITLE
Updates Agile/Continuous-Delivery English & Spanish Guides

### PR DESCRIPTION
**Pull request by jazvillagra**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/agile/continuous-delivery/index.md [hunk 1]
```diff
@@ -7,5 +7,29 @@ Continuous delivery (CD) is a software engineering approach in which teams produ
...
+As a matter of fact, peer-reviewed research shows that this is not the case—high performance teams consistently deliver services faster and more reliably than their low performing competition. This capability provides an incredible competitive advantage for organizations, at ultimately a lower cost in efforts.
 
+### The benefits
+
+The practices at the heart of **CD** will help you achieve a few very important benefits:
+
+* **Low Risk releases.** The primary goal of continuous delivery is to make releases/deployments to production much painless, low-risk events that can be performed at any given time, on demand. By applying certain patterns, it is relatively straightforward to achieve zero-downtime deployments that are undetectable to users.
+
+* **Faster time to market.** When teams work togetherto automate the build and deployment. environment provisioning, and regression testign processes, developers can incorporate integration and regression testing into their daily work and completely remove these phases. Therefore, large amounts of re-work that plague the phased approach can be avoided.
```
Comment (co16353sidak) [inline guide/english/agile/continuous-delivery/index.md]: keep space between `togetherto` -> `together to`
grammatical error: `testign` -> `testing`

File: guide/english/agile/continuous-delivery/index.md [hunk 2]
```diff
@@ -7,5 +7,29 @@ Continuous delivery (CD) is a software engineering approach in which teams produ
...
+
+* **Higher quality.** When developers have automated tools that discover regressions within minutes, teams are freed to focus their effort on user research and higher level testing activities such as exploratory testing, usability testing, and performance and security testing. These activities can be performed continuously throughout the delivery process, ensuring quality in to products and services from the beginning.
```
Comment (co16353sidak) [inline guide/english/agile/continuous-delivery/index.md]: consider removing `to` here
making the line `ensuring quality in products and services from the beginning`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@jazvillagra 3 Suggestions for you to consider:

1. I have a few grammatical suggestions for you to consider in this article.

2. Also on reviewing this, i have found that the content seems very familiar to an external link (listed below), the usual process would be to close this PR but seeing as the content is not a direct attempt to copy, please consider either properly citing the source or referencing the source and then modifying the contents.

3. Also there seems to be a merge conflict of your PR with master, considering removing this, if you get stuck feel free to ask for the help regarding the same.

/cc: @RandellDawson (Second Opinion)

File: guide/english/agile/continuous-delivery/index.md (hunk 2 above)
Comment (s-Hale) [reply guide/english/agile/continuous-delivery/index.md]: Typos in line 21. Periods where there should be commas, "testign". Two 'ands' in line 23. "in to".

File: guide/english/agile/continuous-delivery/index.md [hunk 3]
```diff
@@ -7,5 +7,29 @@ Continuous delivery (CD) is a software engineering approach in which teams produ
 
 It aims at building, testing, and releasing software faster and more frequently. The approach helps reduce the cost, time, and risk of delivering changes by allowing for more incremental updates to applications in production. A straightforward and repeatable deployment process is important for continuous delivery. Continuous delivery means that the team ensures every change can be deployed to production but may choose not to do it, usually due to business reasons.
```
Comment (s-Hale) [inline guide/english/agile/continuous-delivery/index.md]: "It aims to build, test and release..."

**Comment by RandellDawson** [comment]
As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**